### ISSUE TITLE
Finding super-relations without any 'way' or 'node' members in it

### ISSUE THREAD

**Issue by alexander-ii** [drolbr/Overpass-API#470]
Hello and thank you for the Overpass.

I tried to find all relations with tag `public_transport=stop_area_group` in Moscow, Russia and got an empty result. Relations like this particular one https://www.openstreetmap.org/relation/7588556/.

Here is overpass query which i wrote first: https://overpass-turbo.eu/s/xkf
```
[out:xml][timeout:400];
{{geocodeArea:Россия, Москва}}->.searchArea;

relation["public_transport"="stop_area_group"](area.searchArea);

(._;>;); out meta qt;
```

And here is the second one which gave me results that i expected in the first one: https://overpass-turbo.eu/s/xkg
```
[out:xml][timeout:400];
{{geocodeArea:Россия, Москва}}->.searchArea;

  
relation["public_transport"="stop_area"](area.searchArea) -> .sta;
rel["public_transport"="stop_area_group"](br.sta);

(._;._>;); out meta qt;
```

So, am I wrong to expect not empty results in the first query?

**Comment by mmd-osm** [comment]
See https://help.openstreetmap.org/questions/50752/how-do-i-get-route_master-and-its-children-via-overpass-ql-in-overpass-turbo for more details. The description is mentioning bbox, it's basically exactly the same for areas as well.

Duplicate of #293

**Comment by alexander-ii** [comment]
@mmd-osm Thank you for the answer. I get it :)
**tl;dr:** this was made on purpose https://github.com/drolbr/Overpass-API/issues/293#issuecomment-235475013
